src/gpu/intel/jit/conv/plan.cpp

```diff
@@ -37,19 +37,19 @@ namespace jit {
 //
 // | Tile level | Nesting level | Maps to                |
 // |------------|---------------|------------------------|
-// | grid_dim   | 0             | Thread group           |
+// | tg_dim     | 0             | Thread group           |
```

Comment:
There is a mismatch: `tg_dim` at line 74 in this file refers to the size of the thread group while this line refers to the number of thread groups.
We can keep this table as is - it seems to be aligned with the other naming. As an improvement we can change column names:

- "Tile level" to "Extent"
- "Maps to" to "One iteration maps to"
- 
to make it less ambiguate.

Reply:
Done.